build/pkgs/ecl/SPKG.rst

```diff
@@ -43,7 +43,7 @@ Special Update/Build Instructions
 
 -  Do NOT quote SAGE_LOCAL when setting CPPFLAGS and/or LDFLAGS,
    in spkg-install as this caused the build to break. See
-   http://trac.sagemath.org/sage_trac/ticket/10187#comment:117
+   https://github.com/sagemath/sage/issues/10187#comment:117
 
```

Comment:
I think this needs to be changed to an `:issue:`, or Sphinx will complain.

Reply:
see https://github.com/sagemath/sage/actions/runs/4137167886/jobs/7151994051#step:6:21601

Reply:
Waiting to see the results of the docbuild....